lib/util/error.js

```diff
@@ -817,6 +817,13 @@ shaka.util.Error.Code = {
    */
   'INIT_DATA_TRANSFORM_ERROR': 6016,
 
+  /**
+   * The server certificate request failed.  This could be a timeout, a
+   * network failure, or a rejection by the server.
```

Comment:
I'd suggest removing these examples, it gives the impression these are the only possible failures.  I'd suggest just saying `The network request for the server certificate failed` (and keep the `error.data[0]` part).